### PR TITLE
Routing - Creating a Network of payment channels (Language Edits)

### PR DESCRIPTION
**Pull request by ImranLorgat**
Edited the entire section heading above. The example given here is a little bit long-winded and wordy. I tried to make it more concise, but I think it could be streamlined a little further.

Also, there is one corner case I'm not sure of and have let a comment in line.

### CODE REVIEW

File: routing.asciidoc [hunk 1]
```diff
@@ -16,112 +16,180 @@ Finally, you will learn about the SPHINX mix Format that enables onion routing w
...
-_Gloria has 20 hours to show the secrete after the contract was signed. If the time has passed Wei will get his deposit back from the escrow service and the contract becomes invalid._
+[alice-gloria-routing-9]
+----
+_Wei has 20 hours to show the secret after the contract was signed.
+If he does not provide the secret by this time, Bob's deposit will be refunded by the escrow service and the contract becomes invalid._
+----
 
-With such a chain of contracts we can be sure that after 24 hours of setting up the first contract that the payment was either successfully delivered from Alice via Bob and Wei to Gloria or that the payment has failed and was not conducted at all.
+With such a chain of contracts we can ensure that, after 24 hours, the payment will succesfully deliver from Alice to Bob to Wei to Gloria, or it will fail and everyone will be refunded.
 It cannot be stuck in the middle of the road.
```
Comment (ImranLorgat) [inline routing.asciidoc:177]: This part I am not sure about. Consider the corner case:

Gloria reveals the secret to Wei, gets 10 gold coins
Wei reveals the secret to Bob, gets 11 gold coins
Bob's node crashes, goes offline, and he cannot reveal the secret to Alice by the time the HTLC expires.
The HTLC expires, the escrow ends, Alice is refunded her 12 gold coins.

Gloria ends up with 10, Wei ends up with 1, and Alice still has her 12 coins.
Bob paid Wei 11, but did not receive 12 from Alice so he's lost 11 coins and essentially picked up the tab.
What now?

**Comment by ImranLorgat** [comment]
See comment here for the line I'm not sure about

https://github.com/lnbook/lnbook/pull/370/commits/12a43a300fd9345be9f364dffa6b4901bde10068#r477451337

**Comment by aantonop** [comment]
Just fix the conflicts @ImranLorgat please

**Comment by ImranLorgat** [comment]
Probably going to be easier to scrap this and replace with a fresh PR. Will do in the next few days and then close this.

**Comment by aantonop** [comment]
> Probably going to be easier to scrap this and replace with a fresh PR. Will do in the next few days and then close this.

Ok, thanks @ImranLorgat

**Comment by ImranLorgat** [comment]
Scrapped and replaced with #501

**Pull request closed without merging**